src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -1044,15 +1057,20 @@ def __import_kvset_from_file(client, path, yes):
         if __validate_import_config_setting(config_setting):
             kvset_to_import.append(config_setting)
 
+    existing_kvset = __read_kv_from_config_store(client,
+                                                 key=SearchFilterOptions.ANY_KEY,
```

Comment:
>=SearchFilterOptions.ANY_KEY, [](http://example.com/codeflow?start=52&length=29)

Same here, for deleting, we need to match prefix too

Reply:
For raw kv, I think the behavior is to delete all kvs not specified in the import file? @pratiksanglikar

Reply:
Ah yes, you're right. In that case, we need to fetch all keys here, since we don't have any prefix or label filtering for kvset.

Reply:
@shenmuxiaosen yes, that's correct!